doc/source/overview/administration.rst

```diff
@@ -55,66 +55,6 @@ Each repository is expected to follow this minimum set of standards:
 
 Release Procedures and Versioning
 =================================
-PyAnsys library releases are managed through both automated
```

Comment:
All these lines were integrated into the `releasing.rst` file.

Reply:
Even the last few that still show in green? That seems odd.